renderer/reducers/pay.js

```diff
@@ -225,7 +225,7 @@ export const queryRoutes = (pubKey, amount) => async dispatch => {
     })
     dispatch({ type: QUERY_ROUTES_SUCCESS, routes })
   } catch (e) {
-    dispatch({ type: QUERY_ROUTES_FAILURE })
+    dispatch({ type: QUERY_ROUTES_FAILURE, error: e.message })
```

Comment:
Not something that the PR has touched but in 
queryFees error serialization is a bit different.
`const error = get(e, 'response.statusText', e)`
Should probably change to
```
    const error = get(e, 'response.statusText', e.message)
```
there